rasa/nlu/extractors/__init__.py

```diff
@@ -87,3 +87,66 @@ def filter_trainable_entities(
             )
 
         return filtered
+
+
+    @staticmethod
+    def add_roles_to_entities(role_message: Message, message: Message) -> Message:
+       """mark all predicted roles as roles not entities"""
+       starts = {ent["start"]: ent for ent in role_message.get("entities", [])}
+
+       entities = []
+       for ent in message.get("entities", []):
+           start_idx = ent["start"]
+           if start_idx in starts:
+               ent_with_role = ent.copy()
+               ent_with_role["role"] = starts[start_idx]["entity"]
+               entities.append(ent_with_role)
+           else:
```

Comment:
I guess we don't want the else statement. Currently, we get in case no role was detected:
```
  "entities": [
    {
      "start": x,
      "end": y,
      "value": "Buenos Aires",
      "entity": "location",
      "confidence": 0.9792075801341894,
      "extractor": "CRFEntityExtractor"
    }
  ],
...
  "roles": [
    {
      "start": x,
      "end": y,
      "value": "Buenos Aires",
      "entity": "location",
      "confidence": 0.9792075801341894,
      "extractor": "CRFEntityExtractor"
    }
  ],
```
The roles is just a duplicate of entities. Shouldn't roles only contain "entites" that are actual roles?